client/src/components/Day.jsx

```diff
@@ -17,14 +17,30 @@ class Day extends React.Component {
     }));
   }
 
+  // Helper function that changes background of button when user's mouse enters button area
+  changeBackgroundEnteringButton(e) {
+    const { clicked } = this.state;
```

Comment:
If your event is entering an area, might want to use a related name for the state.